src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -219,6 +219,47 @@ StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinat
 StopPlace/TransportMode | | | use the mode of __highest priority__ (see [NTFS specifications](https://github.com/CanalTP/ntfs-specification/blob/v0.11.1/ntfs_fr.md#physical_modestxt-requis))
 StopPlace/StopPlaceType | | | see the section [StopPlaceType mapping](#stopplacetype-mapping)
 
+## correspondances.xml
+Each connection between two stops in `transfers.txt` produces a `SiteConnection` element with the `From` and `To` nodes of the connection as well as a `WalkTransferDuration` node.
+All `SiteConnection` elements are grouped in a `members` element inside a `GeneralFrame`.
+
+### Top level structure
+Example:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame id="FR:GeneralFrame:NETEX_RESEAU:" version="any">
+	<TypeOfFrameRef ref="FR:TypeOfFrame:NETEX_RESEAU:"/>
```

Comment:
I would remove that for now to stay homogeneous with the rest of the specification.